spec/puppet-lint/configuration_spec.rb

```diff
@@ -58,9 +61,25 @@
       'log_format'       => '',
       'with_context'     => false,
       'fix'              => false,
+      'github_actions'   => false,
       'show_ignored'     => false,
       'json'             => false,
       'ignore_paths'     => ['vendor/**/*.pp']
     )
   end
+
+  it 'detects github actions' do
+    override_env do
+      ENV['GIHUB_ACTION'] = 'action'
+      subject.defaults
+    end
+  end
+
+  def override_env
+    old_env = ENV.clone
+    yield
+  ensure
+    ENV.clear
+    ENV.update(old_env)
+  end
```

Comment:
I wasn't quite sure about this, but I wrote it on the train without internet access.

Reply:
Is this comment resolvable?

Reply:
Well, it works. I just don't know if this is the desired solution in this repo.